fakecreds.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"os"
+)
+
+// Create a fake creds.json file for testing
+func CreateFakeCreds(creds string) (err error) {
+	content := "{\n  \"type\": \"service_account\"," +
```

Comment:
create a test file called test_creds.json under testdata folder and use that in main_test.go

Reply:
fixed